python/llm/example/CPU/HF-Transformers-AutoModels/Streaming/README.md

```diff
@@ -0,0 +1,39 @@
+# BigDL-LLM Transformers Low-Bit Efficient Streaming Large Language Models with Attention Sinks
+
+There are two major challenges to deploy Large Language Models (LLMs) in streaming applications where long interactions are expected. 
```

Comment:
Instead of copying descriptions from https://github.com/mit-han-lab/streaming-llm, simply say that we apply low-bit optimizations to streaming-llm using BigDL-LLM, which deploy low-bit LLMs for infinite-length inputs.